### PR TITLE
RHIROS-512  ROS CTA

### PR DESCRIPTION
**Pull request by r14chandra**
Payload for no. of systems greater than zero
Screenshot from 2022-03-03 15-58-36

Payload for no. of systems equals to zero 
Screenshot from 2022-03-03 15-12-19

### CODE REVIEW

File: ros/api/v1/call_to_action.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                    "redhatInsights": [
+                        {
+                            "id": "ros-1",
+                            "description": "Resource optimization recommends to assess and monitor"
+                            + " cloud usage and optimization on these systems",
+                            "icon": "cog",
+                            "action": {
+                                "title": f"View {total_system_count} {suffix}"
+                                + " with suggestions",
+                                "href": "/insights/ros"
```
Comment (kgaikwad) [inline ros/api/v1/call_to_action.py:53]:
```suggestion
                                "href": "/insights/ros?with_suggestions=true"
```
As in title, it talks about how many systems with recommentations then will it be good idea to redirect to page where it displays the number of systems with suggestions?

**Review by kgaikwad — CHANGES_REQUESTED** [review]
Thank you @r14chandra! 
Overall looks good! Added small inline comment.

File: ros/api/v1/call_to_action.py (hunk 1 above)
Comment (r14chandra) [reply ros/api/v1/call_to_action.py]: Thanks @kgaikwad. Pushed the change 👍🏻

**Review by kgaikwad — APPROVED** [review]
LGTM :+1:

**Pull request merged**